captcha/templates/captcha/includes/js_v3.html

```diff
@@ -6,7 +6,7 @@
         .then(function(token) {
             console.log("reCAPTCHA validated for 'data-widget-uuid=\"{{ widget_uuid }}\"'. Setting input value...")
             var element = document.querySelector('.g-recaptcha[data-widget-uuid="{{ widget_uuid }}"]');
-            element.value = token;
+            element.value = token || "-";
```

Comment:
it would be nice if we could create a new template tag that grabs whether this is in DEBUG mode or not.